palantirnet/search_api_federated_solr
# UMUS-128 pull new version (D8)

This uses the work from https://github.com/palantirnet/federated-search-react/pull/4 to pull the dependency from the new private repo.

To test:
- Checkout this branch at `src/search_api_federated_solr`
- `drush @healthblog.consumer.local cr`
- https://healthblog.uofmhealth.local/search-app?search=cancer see that the app works as expected.
